charts/testkube-logs-service/values.yaml

```diff
@@ -0,0 +1,197 @@
+# Default values for testkube-cloud-api.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+global:
+  # -- Domain under which to create Ingress rules
+  domain: ""
+  # -- UI subdomain which get prepended to the domain
+  uiSubdomain: "cloud"
+  # -- REST API subdomain which get prepended to the domain
+  restApiSubdomain: "api"
+  # -- gRPC API subdomain which get prepended to the domain
+  grpcApiSubdomain: "agent"
+  # -- Websocket API subdomain which get prepended to the domain
+  websocketApiSubdomain: "websockets"
+  # -- TLS provider (possible values: "", "cert-manager")
+  certificateProvider: ""
+  certManager:
+    # -- Certificate Issuer ref (only used if `provider` is set to `cert-manager`)
+    issuerRef: ""
+  # -- Global image pull secrets (provided usually by a parent chart like testkube-enterprise)
+  imagePullSecrets: []
+  ingress:
+    # -- Toggle whether to enable or disable all Ingress resources (if false, all Ingress resources will be disabled and cannot be overriden)
+    enabled: true
+
+replicaCount: 1
+image:
+  repository: kubeshop/testkube-log-service
+  pullPolicy: IfNotPresent
+  tag: "1.6.7"
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: "logs-service"
+# -- Additional env vars to be added to the deployment
+additionalEnv: {}
+# FOO: bar
+
+ingress:
+  # -- Common annotations which will be added to all Ingress resources
+  annotations:
+    nginx.ingress.kubernetes.io/force-ssl-redirect: "true"
+    nginx.ingress.kubernetes.io/preserve-trailing-slash: "true"
+  # -- Common Ingress class name (NGINX is the only officially supported ingress controller and should not be changed)
+  className: nginx
+
+restIngress:
+  # -- Toggle whether to enable the REST API Ingress
+  enabled: true
+  # -- Additional labels to add to the REST Ingress resource
+  labels: {}
+  # -- Additional annotations to add to the REST Ingress resource
+  annotations: {}
+  # kubernetes.io/ingress.class: nginx
+  # -- Hostname for which to create rules and TLS certificates (if omitted, the host will be generated using the global subdomain and `domain` values)
+  host: ""
+
+grpcIngress:
+  # -- Toggle whether to enable the gRPC API Ingress
+  enabled: true
+  # -- Additional labels to add to the gRPC Ingress resource
+  labels: {}
+  # -- Additional annotations to add to the gRPC Ingress resource
+  annotations: {}
+  # kubernetes.io/ingress.class: nginx
+  # -- Max payload size for proxied gRPC API
+  maxPayloadSize: 16m
+  # -- Hostname for which to create rules and TLS certificates (if omitted, the host will be generated using the global subdomain and `domain` values)
+  host: ""
+
+websocketsIngress:
```

Comment:
I think there's no need for `websocketsIngress` and `restIngress` here, as you only create `grpcIngress`